src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -141,13 +146,14 @@
     private transient String id;
 
     @DataBoundConstructor
-    public LDAPConfiguration(@NonNull String server, String rootDN, boolean inhibitInferRootDN, String managerDN, Secret managerPasswordSecret) {
+    public LDAPConfiguration(@NonNull String server, boolean sslVerify, String rootDN, boolean inhibitInferRootDN, String managerDN, Secret managerPasswordSecret) {
```

Comment:
The old constructor signature should be kept but deprecated, calling the new signature with the default value for the new field.

Reply:
New commit resolved. Please help review.